pkg/extensions/mgmt.md

```diff
@@ -0,0 +1,46 @@
+# `mgmt`
+
+`mgmt` component provides an endpoint for configuration management
+
+Response depends on the user privileges:
+- unauthenticated and authenticated users will get a stripped config
+- admins will get full configuration with passwords hidden (not implemented yet)
+
+
+| Supported queries | Input | Output | Description |
+| --- | --- | --- | --- |
+| [Get current configuration](#get-current-configuration) | None | config json | Get current zot configuration | 
+
+
+## Get current configuration
+
+**Sample request**
+
+```bash
+curl http://localhost:8080/v2/_zot/ext/mgmt | jq
+```
+
+**Sample response**
+
+```json
+{
+  "distSpecVersion": "1.1.0-dev",
+  "GoVersion": "go1.19.2",
+  "Commit": "v1.1.15-rc1-85-gae8ae7d-dirty",
+  "ReleaseTag": "v1.1.15-rc1",
+  "BinaryType": "-sync-search-scrub-metrics-lint-mgmt",
+  "HTTP": {
+    "Auth": {
+      "FailDelay": 0,
+      "HTPasswd": {
+        "Path": "test/data/htpasswd"
+      },
+      "LDAP": {
+        "Address": "ldap.example.org"
```

Comment:
This behavior was changed in the last amends, right?

Reply:
oh, yes, will fix